common/src/main/java/io/netty/util/internal/logging/Slf4JLoggerFactory.java

```diff
@@ -40,31 +37,8 @@ public Slf4JLoggerFactory() {
 
     Slf4JLoggerFactory(boolean failIfNOP) {
         assert failIfNOP; // Should be always called with true.
-
-        // SFL4J writes it error messages to System.err. Capture them so that the user does not see such a message on
```

Comment:
I think it does more harm then any good... So I think we should jus remove it. Thats why I added @trustin as reviewer as he wrote the code ;)